.circleci/config.yml

```
@@ -471,12 +478,17 @@ workflows:
       - test_horizon_integration:
           name: test_horizon_integration_with_captive_core
           enable-captive-core: true
-      - test_verify_range_docker_image
+      - test_verify_range_docker_image:
+          # test pusblishing
+          publish: true
```

We don't want to publish on non-master, correct?
```suggestion
          publish: false
```

---

I'm testing that it works before merging

---

But ... it seems like the secrets are not there (probably due to this being a PR from my fork).

I will revert it and test from `master`